kubevirt/add-services/add-services.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: k6t-add-services
+  annotations:
+    policies.kyverno.io/title: Add Services
+    policies.kyverno.io/category: KubeVirt
+    policies.kyverno.io/subject: VirtualMachine
+    policies.kyverno.io/description: >-
+      Add an SSH Service to every VirtualMachineInstance which is getting created.
+      This Service will use a ClusterIP, thus the admin has to ensure that the IP
+      space is large enough and ClusterIP type can be met.
+
+spec:
+  background: false
+  rules:
+  - name: k6t-add-service-ssh
+    match:
+      any:
+      - resources:
+          kinds:
+          - VirtualMachineInstance
+    generate:
+      apiVersion: v1
+      kind: Service
+      name: "{{request.object.metadata.name}}"
+      namespace: "{{request.object.metadata.namespace}}"
+      synchronize: true
+      data:
+        metadata:
+          ownerReferences:
+          - apiVersion: kubevirt.io/v1
+            kind: VirtualMachineInstance
+            name: "{{request.object.metadata.name}}"
+            uid: "{{request.object.metadata.uid}}"
+        spec:
```

Comment:
Just a suggestion, but I might force the type `ClusterIP` here in my definition so it's crystal clear what will be created without relying upon field/value defaults of the API server.

Reply:
Agreed. Thanks for weighing in here.